account/guides/setupRegistrar.md

```diff
@@ -0,0 +1,147 @@
+| `Realm` | optional | String identifying the realm the user belongs.  If left blank/empty the default realm on the account will be used. |
+| `HttpVoiceV2AppId` | optional | String identifying the V2 Voice application to route the call to. |
+| `Hash1`    | required | String representing a potential Hash values used to authenticate the client. The value should be computed from an MD5 Hash of: {composite-username}:{Realm}:{Password}. |
+| `Hash2`   | required | String representing a potential Hash value used to authenticate the client. The value should be computed from an MD5 Hash of {composite-username}:{Realm}:{Realm}:{Password}. |
+
+The Twilio platform requires a `SIPAuthUsername` and a `SIPAuthPassword`. These examples assume that your `username=sipauthtest` and `password=password`.
+
+### Register Device {#register-sip-device}
+After the realm and associated Sip Credential has been provisioned within the Bandwidth Dashboard, input that information into a registered device to begin sending registrations and place SIP calls.
+
+### Generate MD5 Hash {#generate-md5-hash}
+Either using the command line or an [online tool](http://www.miraclesalad.com/webtools/md5.php) generate the md5 hash from the username and desired password.
+
+Most *nix (mac, linux) come with MD5 hash built in. Check that MD5 is installed by opening up the terminal and typing:
+Check that MD5 is installed
+
+```bash
+$ which md5
+/sbin/md5
+```
+
+Once MD5 is installed, run the command: `md5 -s {composite-username}:{Realm}:{Password}` where `{Password}` is the desired password
+
+#### Default Setup (_No Domain Specified as Part of the User's ID_)
+##### Generate MD5 `Hash1` _without_ Domain
+| composite-username | : | realm                       | : | password   |
+|--------------------|---|-----------------------------|---|------------|
+| `sipauthtest`      | : | `<randomAccountHex>.auth.bandwidth.com` | : | `password` |
+```
+md5 -s sipauthtest:<randomAccountHex>.auth.bandwidth.com:password
+MD5 ("sipauthtest:<randomAccountHex>.auth.bandwidth.com:password") = fe438bddfc087dda89d29e637f5684ab
+```
+
+##### Generate MD5 `Hash2` _without_ Domain
+| composite-username | : | realm                       | realm                       | : | password   |
+|--------------------|---|-----------------------------|-----------------------------|---|------------|
+| `sipauthtest`      | : | `<randomAccountHex>.auth.bandwidth.com` | `<randomAccountHex>.auth.bandwidth.com` | : | `password` |
+```bash
+md5 -s sipauthtest:<randomAccountHex>.auth.bandwidth.com:<randomAccountHex>.auth.bandwidth.com:password
+MD5 ("sipauthtest:<randomAccountHex>.auth.bandwidth.com:<randomAccountHex>.auth.bandwidth.com:password") = 79bb0e55551e14a2f329a282c7cf145
```

Comment:
I think the method for generating hash1 is right, but I believe we need to change this and other mentions of generating hash2 to use `username@realm:realm:password`.

Reply:
So in this example it would change to `sipauthtest@<randomAccountHex>.auth.bandwidth.com:<randomAccountHex>.auth.bandwidth.com:password`?

Reply:
Yeah that looks right to me

Reply:
Done